BUILDING.md

```diff
@@ -0,0 +1,39 @@
+# Building Protocol Buffers
+
+These are instructions for building the open source release of protobuf from source.
+
+Given the right tools, this project can be built on Linux, Mac OS X, or Windows though Linux is by far the most commonly used and best tested platform.
+
+## Prerequisites
+
+Likely incomplete
+
+* git
```

Comment:
It's been a while since I completely built from scratch. Maybe you could get a member of the team to do this to make sure the pre-requisites are correct?

Reply:
Ok, just responding to "Likely incomplete"